wpilibc/src/main/native/include/frc/drive/DifferentialDrive.h

```diff
@@ -152,10 +152,10 @@ class DifferentialDrive : public RobotDriveBase {
    *                      [-1.0..1.0]. Forward is positive.
    * @param rightSpeed    The robot right side's speed along the X axis
    *                      [-1.0..1.0]. Forward is positive.
-   * @param squaredInputs If set, decreases the input sensitivity at low speeds.
+   * @param squareInputs If set, decreases the input sensitivity at low speeds.
    */
   void TankDrive(double leftSpeed, double rightSpeed,
-                 bool squaredInputs = true);
+                 bool squareInputs = true);
```

Comment:
This should be pulled up to the previous line (see azure build wpiformat job).

Reply:
What are the chances that it was an 81 character long line before I made changes... fixed now.